src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -27,36 +31,40 @@ class ReviewPage extends React.Component {
     focusElement('h2');
   }
 
+  renderDowntime = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return <Message downtime={downtime} />;
+    }
+
+    return children;
+  };
```

Comment:
Copied from `RoutedSavableReviewPage`, with the exception that _this_  `DowntimeMessage` isn't the `save-in-progress` version.

https://github.com/department-of-veterans-affairs/vets-website/blob/d412bc0a1834ff507761f4be7f4d94eebb876349/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L58-L66